src/components/generic/TimerComps.tsx

```diff
@@ -0,0 +1,289 @@
+import { useContext } from 'react';
+import { RemoveButtonStyle } from '../../utils/Styled';
+import { type Timer, TimersContext } from '../context/TimersContextProvider';
+import Countdown from '../timers/Countdown';
+import Stopwatch from '../timers/Stopwatch';
+import Tabata from '../timers/Tabata';
+import XY from '../timers/XY';
+
+/*
+ *
+ */
+type TimeDisplayProps = {
+    value: string;
+    label?: string;
+};
+
+export const TimeDisplay = ({ value, label }: TimeDisplayProps) => {
+    return (
+        <h1 className="text-2xl font text-center p-1">
+            {label !== undefined ? `${label}: ` : ''}
```

Comment:
here you can use `label` directly, as `undefined` will not render anything in React